src/loaders/ttf/tvgTtfLoader.h

```diff
@@ -37,12 +37,15 @@ struct TtfLoader : public FontLoader
     char* text = nullptr;
     Shape* shape = nullptr;
     bool italic = false;
+    bool fromMemory = false;
```

Comment:
Let's align the variable name.  (fromMemory -> freeData)
![image](https://github.com/thorvg/thorvg/assets/3711518/60df0af4-0f02-4b72-b11f-6a05268f0143)